projectgenerator/gui/import_data.py

```diff
@@ -392,7 +391,9 @@ def help_requested(self):
                 "https://opengisch.github.io/projectgenerator/docs/user-guide.html#import-an-interlis-transfer-file-xtf")
 
     def advance_progress_bar_by_text(self, text):
-        if text.strip() == 'Info: compile models…':
```

Comment:
Is there something wrong with using the ellipsis?

Reply:
Of course there is...

Reply:
It wasn't fulfilling the condition, that is, the progress bar was never showing the 50% and 75% indicators. Could you please confirm if the ellipsis actually works for you?